.github/workflows/runE2ETest.yml

```diff
@@ -29,6 +33,17 @@ jobs:
         with:
           path: ./salesforcedx-vscode
           ref: ${{ github.event.ref }}
+      - name: Download extension vsixes
+        run: |
+          mkdir ./extensions
+          pwd
+          gh run download ${{ inputs.runId }} -D ./extensions
```

Comment:
Is runId a GHA thing or did we create the term?

Reply:
Just calling out for future us using gh here means the run with the artifacts has to have already been completed.